summit/2017.html

```diff
@@ -1,6 +1,6 @@
 ---
 layout: default
-permalink: /summit/
+permalink: /summit/2017
```

Comment:
I don't think this is enough in order to archive the 2017 version. You've changed the styling in _summit.scss so it won't be preserved.

Reply:
True, my bad; I can copy the old _summit.scss and use that for the /2017 page, or I can change the div ID-s/classes so it looks the same.

Reply:
Yes, simply archive it as summit-old.scss :)

Reply:
Yes, simply archive it as summit-old.scss :)